general_rules/PenaltiesBonuses.tex

```diff
@@ -9,22 +9,25 @@ \subsection{Penalty for not attending}\label{rule:not_attending}
 \begin{enumerate}
 	\item \textbf{Automatic schedule:} All teams are automatically scheduled for all tests.
 	\item \textbf{Announcement:} If a team cannot participate in a test (for any reason), the team leader has to announce this to the OC at least \timing{60 minutes} before the test slot begins.
-	\item \textbf{Penalties:} A team that is not present at the start position when their scheduled test starts, the team is not allowed to participate in the test anymore. If the team has not announced that it is not going to participate, it gets a penalty of \scoring{\penaltybig points}. 
+	\item \textbf{Penalties:} A team that is not present at the start position when their scheduled test starts, the team is not allowed to participate in the test anymore. If the team has not announced that it is not going to participate, it gets a penalty of \scoring{\penaltybig points}.
 \end{enumerate}
 
 \subsection{Extraordinary penalties}\label{rule:extraordinary_penalties}
 \begin{enumerate}
-	\item \textbf{Penalty for inoperative robots:} If a team starts a test, but it does not solve any of the partial tasks (and is obviously not trying to do so), a penalty of \scoring{\penaltysmall points} is handed out. The decision is made by the referees and the monitoring TC member.  
+	\item \textbf{Penalty for faking robots:} If a team starts a test, but it does not solve any of the partial tasks (and is obviously not trying to do so), a penalty of \scoring{\penaltybig points} is handed out. The decision is made by the referees and the monitoring TC member.
```

Comment:
Part 2; Most significant change here: make the penalty a big one (150 points with this PR)